src/cfml/system/services/ServerService.cfc

```diff
@@ -8,7 +8,7 @@
 * I manage servers
 *
 */
-component accessors="true" singleton {
+component accessors="true" singleton extends="commandbox.system.BaseCommand" {
```

Comment:
@mgmathus  Why have you added this?  The `ServerService` is not a command, nor should it be extending the basecommand